kuryr_kubernetes/cni/prometheus_exporter.py

```diff
@@ -46,6 +46,9 @@ def metrics(self):
         return flask.Response(collected_metric, mimetype='text/plain')
 
     def run(self):
+        # Disable obtrusive werkzeug logs.
+        logging.getLogger('werkzeug').setLevel(logging.WARNING)
```

Comment:
I'm wondering if it should go in the daemon main process[1],  we had issues when disabling the pyroute2 WARNING on the other children processes .

[1] https://github.com/openshift/kuryr-kubernetes/blob/master/kuryr_kubernetes/cni/daemon/service.py#L399

Reply:
That was a problem with registry creds BTW. ;)